pages/PopCard.js

```diff
@@ -0,0 +1,46 @@
+import { useRouter } from "next/router";
+import Image from "next/image";
+import useSWR from "swr";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
+
+export default function PopCard({ name, location, image, bewerten, preis }) {
```

Comment:
versuche in der Sprache einheitlich zu bleiben :), also Englisch!
Aber das destructuring ist sowieso nicht nötig, weil du es gar nicht verwendest?
```suggestion
export default function PopCard() {
```